src/pages/projectCreate/ProjectCreationWalletConnectionForm.tsx

```diff
@@ -184,7 +189,7 @@ export const ProjectCreationWalletConnectionForm = ({
                     <WalletConnectionDetails
                       projectWallet={{
                         connectionDetails: {
-                          grpcPort: Number(nodeInput.grpc),
+                          grpcPort: nodeInput.isVoltage ? 10009 : Number(nodeInput.grpc),
```

Comment:
If the nodeInput.isVoltage is present, it already means the nodeInput.grpc would already be 10009 no? What difference did this change make?

Reply:
The difference is that it will display 10009 at the grpc port field instead of the current 0
<img width="614" alt="Screenshot 2024-04-02 at 6 26 06 PM" src="https://github.com/geyserfund/geyser-app/assets/29054481/8d690fcc-9a3d-4288-ae42-bae2f40faff2">

Reply:
Love that, you noticed it and corrected it. Good job.